js/localization.js

```diff
@@ -10,7 +10,7 @@ const i18nextXHRBackend = require('i18next-xhr-backend');
 const Store = require('electron-store');
 const store = new Store();
 
-const availableLanguages = ['en'];
+const availableLanguages = ['en' , 'el'];
```

Comment:
We should probably auto populate this list.

Not a change you need to make, but an acknowledgement that this is not ideal.